### ISSUE TITLE
Default to iCloud when Dropbox is not installed

### ISSUE THREAD

**Issue by yangm97** [lra/mackup#1109]
As title says. So we don’t need to manually copy the mackup settings from iCloud before using it.

**Comment by stvhwrd** [comment]
Piggybacking on this issue - a default to iCloud (i.e. looking for `~/Library/Mobile Documents/com~apple~CloudDocs/Mackup`) in the absence of a `~/.mackup.cfg` config file would make sense.

If the user **has** specified a storage engine in the config file, then that should be respected regardless of whether Dropbox, Google Drive, etc. are installed.

**Comment by edenvicary** [comment]
I totally agree, iCloud comes default and for a project titled mac-kup it would make sense to default to the most ubiquitous platform on the system, even if it's not people's day to day main file storage.
